KK_ButtPlugin/ButtPlugin.cs

```diff
@@ -28,21 +30,35 @@ private void Start()
                 defaultValue: "ws://localhost:12345/",
                 "The Buttplug server address (requires game restart).");
             MaxStrokesPerMinute = Config.Bind(
-                section: "Device",
+                section: "Stroker Settings",
                 key: "Maximum strokes per minute",
                 defaultValue: 140,
                 "The top speed possible on your stroker at 70% stroke length.");
             LatencyMs = Config.Bind(
-                section: "Device",
+                section: "Stroker Settings",
                 key: "Latency (ms)",
                 defaultValue: 0,
                 "The difference in latency between your stroker and your display. \n" +
                 "Negative if your stroker has lower latency.");
             EnableVibrate = Config.Bind(
-                section: "Device",
+                section: "Vibration Settings",
                 key: "Enable Vibrators",
                 defaultValue: true,
-                "Maps control speed to vibrations");
+                "Maps control speed to vibrations"
+            );
+            SyncVibrationWithAnimation = Config.Bind(
+                section: "Vibration Settings",
+                key: "Vibration With Animation",
+                defaultValue: false,
+                "Maps vibrations to a wave pattern in sync with animations.\n" +
+                "Timings are approximations based on animation length and not precise location of stimulation."
+            );
+            VibrationFrequency = Config.Bind(

```

Comment:
Name is a tad confusing to me. This is not the frequency OF the vibration. How about VibratorUpdateFrequency? That's a mouthful but more descriptive IMO.